.github/workflows/lib-injection.yml

```diff
@@ -0,0 +1,77 @@
+name: "Library Injection"
+on:
+  push:
+    branches:
+      - 1.x
+  pull_request:
+    branches:
+      - 1.x
+
+jobs:
+  build-and-publish-init-image:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/1.x'
```

Comment:
Is this required? This action should only be triggered on pull requests targeting 1.x and pushes to 1.x

Reply:
it's just another guard on the version in case the workflow triggers have to change